ext/mysqli/tests/mysqli_query_unicode.phpt

```diff
@@ -4,12 +4,10 @@ mysqli_query() - unicode (cyrillic)
 mysqli
 --SKIPIF--
 <?php
-require_once 'skipifconnectfailure.inc';
-require_once 'table.inc';
+require_once __DIR__ . '/test_setup/test_helpers.inc';
+$link = mysqli_connect_or_skip();
 if (!$res = mysqli_query($link, "SHOW CHARACTER SET LIKE 'utf8'"))
     die("skip UTF8 chatset seems not available");
```

Comment:
This seems wrong. This should either be fixed or simply removed. If I recall correctly, `utf8` is still available in all major providers. Judging by how we never had a problem with this check not working, I'd assume it's safe to remove it for now.

Reply:
Indeed, I'm going to remove.